libs/env/index.js

```diff
@@ -1,12 +1,16 @@
 import fs from "node:fs";
 import path from "node:path";
-import { cwd } from "node:process";
 
 import dotenv from "dotenv";
 
 import { VALID_FLAW_CHECKS } from "../constants/index.js";
 
-export const ROOT = path.join(cwd(), ".");
+const currentDir = new URL(import.meta.url).pathname
+  .slice(process.platform === "win32" ? 1 : 0) // remove leading slash on windows
+  .split("/")
+  .slice(0, -1)
+  .join(path.sep);
+export const ROOT = path.join(currentDir, "..", "..");
```

Comment:
should be a revert of https://github.com/mdn/yari/commit/d3f7ec15aeb45145849683116ccda7455897ffa1#diff-309d7c0018f34c8bb48de097e4b860c64e2a2bb987dbce04f88f10a2bb7cb559L9